lib/stat_tracker.rb

```diff
@@ -3,21 +3,23 @@
 require_relative './game_teams_collection'
 require 'csv'
 
-#THE PARENT
 class StatTracker
   attr_reader :games, :teams, :game_teams
 
   def initialize(locations)
-    @games = GamesCollection.new(locations[:games], self)
     @teams = TeamsCollection.new(locations[:teams], self)
+    @games = GamesCollection.new(locations[:games], self)
     @game_teams = GameTeamsCollection.new(locations[:game_teams], self)
-    # @seasons_collection = SeasonsCollection.new(self)
   end
 
   def self.from_csv(locations)
     self.new(locations)
   end
 
+  def find_season_id(id)
+    @games.find_season_id(id)
+  end
+
   def find_by_id(id)
     @teams.find_by_id(id) || @games.find_by_id(id)
   end
```

Comment:
It's cool that this works but Mike did not like it so I think we're probably better off creating separate find_team_name and find_season methods